src/Symfony/Component/DependencyInjection/Tests/Compiler/CheckExceptionOnInvalidReferenceBehaviorPassTest.php

```diff
@@ -88,6 +89,22 @@ public function testProcessThrowsExceptionOnNonSharedUninitializedReference()
         $this->process($container);
     }
 
+    public function testProcessDefinitionWithBindings()
+    {
+        $container = new ContainerBuilder();
+
+        $container->register('a', 'stdClass');
```

Comment:
this test will now also pass without your change I believe? we should remove this so the Reference `a` in the bindings does actually not exist on the container?

Reply:
oops :) fixed